code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/impl/GithubOAuthImpl.java

```diff
@@ -74,50 +85,15 @@ public Account getAccount(ApiRequest request) {
             return null;
         }
 
-        Account projectAccount = authDao.getAccountById(new Long(unobfuscatedId));
-
-        if (validateProjectAccount(projectAccount, token)) {
-            return projectAccount;
+        project = authDao.getAccountById(new Long(unobfuscatedId));
```

Comment:
There is a chance but only if someone were to really be trying to break the system. I will throw a null check here, and an appropriate error.